fuel/transformers/__init__.py

```diff
@@ -590,3 +590,31 @@ def get_data(self, request=None):
         if data == StopIteration:
             raise StopIteration
         return data
+
+
+class RenameStream(Transformer):
```

Comment:
You did exactly as the ticket suggested, but I'd change the name to simply `Rename`. The `Stream` suffix looks redundant.